gr-blocks/lib/message_strobe_impl.cc

```diff
@@ -91,7 +91,7 @@ namespace gr {
     void message_strobe_impl::run()
     {
       while(!d_finished) {
-        boost::this_thread::sleep(boost::posix_time::milliseconds(d_period_ms));
+        boost::this_thread::sleep(boost::posix_time::milliseconds(static_cast<long>(d_period_ms)));
```

Comment:
This is already in master too: https://github.com/gnuradio/gnuradio/commit/42ea667f9ae27e377a010df7050b203276412d2b .